client/components/Story.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+import { StyledDesc, NavHeader, SectionHeader, Caption, Img } from '../StyledDiv.js';
+
+const Story = (props) => {
+  const {story} = props;
+  const { textTop, imageMiddle, imageMiddleCaption, textBottom } = story;
+  return (
+    <StyledDesc>
+      <NavHeader>Story</NavHeader>
+      <div>
+        <p>{textTop.replace(/[\r]+/g, '\n')}</p>
```

Comment:
Everything looks good. The only thing I would do differently is to repopulate the database so that this type of parsing isn't necessary, which I think is what you mentioned in the stand up. Other than that everything looks really clean. Nice work!

Reply:
Oh thanks for mentioning that. I just removed that and it looks like with CSS it's not even required anymore.